_articles/example-tutorial.md

```diff
@@ -1,84 +0,0 @@
----
```

Comment:
Could we somehow keep this as reference for someone that doesn't use markdown? Maybe merge with contributing.md

Reply:
I have actually added a small note about markdown syntax: [`248c105`#diff-19772ee6e6b148f4f09d6ca8406b36faR58](https://github.com/ModDota/moddota.github.io/pull/27/commits/248c105b584dd841ebcb30daeb1ca08047ab1e50#diff-19772ee6e6b148f4f09d6ca8406b36faR58). I don't see much point in having a full markdown guide here when there is already a lot of existing ones. Though maybe I should add some notes about headings, because there are some specifics about using it in Docusaurus.